var/spack/repos/builtin/packages/albany/package.py

```diff
@@ -0,0 +1,128 @@
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Albany(CMakePackage):
+    """Albany is an implicit, unstructured grid, finite element code for the
+       solution and analysis of multiphysics problems.  The Albany repository
+       on the GitHub site contains hundreds of regression tests and examples
+       that demonstrate the code's capabilities on a wide variety of problems
+       including fluid mechanics, solid mechanics (elasticity and plasticity),
+       ice-sheet flow, quantum device modeling, and many other applications."""
+
+    # Add a proper url for your package's homepage here.
+    homepage = "http://gahansen.github.io/Albany"
+    url      = "https://github.com/gahansen/Albany/tarball/master"
+
+    maintainers = ['gahanse']
+
+    # Add proper versions and checksums here.
+    version('master', git='https://github.com/gahansen/Albany.git', branch='master')
+
+    variant('install',          default=True,
+            description='Enable Install')
+    variant('lcm',          default=True,
+            description='Enable LCM')
+    variant('aeras',          default=False,
+            description='Enable AERAS')
+    variant('qcad',          default=False,
+            description='Enable QCAD')
+    variant('hydride',          default=False,
+            description='Enable HYDRIDE')
+    variant('lcm_spec',          default=False,
+            description='Enable LCM_SPECULATIVE')
+    variant('lame',          default=False,
+            description='Enable LAME')
+    variant('edebug',          default=False,
+            description='Enable DEBUGGING')
+    variant('fpe',          default=False,
+            description='Enable CHECK_FPE')
+    variant('scorec',          default=False,
+            description='Enable SCOREC')
+    variant('felix',          default=False,
+            description='Enable FELIX')
+    variant('mor',          default=False,
+            description='Enable MOR')
+    variant('ci',          default=False,
+            description='Enable ALBANY_CI')
+    variant('ascr',          default=False,
+            description='Enable ALBANY_ASCR')
+    variant('perf',          default=False,
+            description='Enable PERFORMANCE_TESTS')
+    variant('64bit',          default=True,
+            description='Enable 64BIT')
+
+    # Add dependencies
+    depends_on('mpi')
+    depends_on('trilinos@develop:~superlu-dist+isorropia+tempus+rythmos+teko+intrepid+intrepid2+minitensor+phal+pnetcdf+nox+piro+rol+shards+stk+superlu')
+    depends_on('trilinos@develop:~superlu-dist+isorropia+tempus+rythmos+teko+intrepid+intrepid2+minitensor+phal+pnetcdf+nox+piro+rol+shards+stk+superlu', when='+lcm')
```

Comment:
why do you need this one if it's the same as the previous line?

Reply:
I'll fix...